### PR TITLE
fix #953, attribute error in pronunciation assessment python sample

### LINKED ISSUE #953: pronunciation assessment python sample failed with AttributeError
**Describe the bug**

pronunciation assessment python sample failed with
 `reference_words =reference_text.lower().split().strip(string.punctuation)
AttributeError: 'list' object has no attribute 'strip'
`

**To Reproduce**
run python sample, try `pronunciation_assessment_continuous_from_file`

**Expected behavior**

**Version of the Cognitive Services Speech SDK**
1.15

**Platform, Operating System, and Programming Language**
 - Windows, Python

**Additional context**
 - Error messages, log information, stack trace, ...
 - If you report an error for a specific service interaction, please report the SessionId and time (incl. timezone) of the reported incidents. The SessionId is reported in all call-backs/events you receive.
 - Any other additional information

### PR DESCRIPTION
**Pull request by yulin-li**
## Purpose

* ...

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: samples/python/console/speech_sample.py [hunk 1]
```diff
@@ -700,7 +700,7 @@ def recognized(evt):
     # For continuous pronunciation assessment mode, the service won't return the words with `Insertion` or `Omission` even miscue is enabled.
```
Comment (BrianMouncer) [inline samples/python/console/speech_sample.py:700]: >even miscue is enabled. [](start = 118, length = 23)

"even if miscue is enabled"?

**Review by BrianMouncer — APPROVED** [review]
:shipit:

File: samples/python/console/speech_sample.py [hunk 2]
```diff
@@ -700,7 +700,7 @@ def recognized(evt):
...
     # We need to compare with the reference text after received all recognized words to get these error words.
     if enable_miscue:
-        reference_words = reference_text.lower().split().strip(string.punctuation)
+        reference_words = [w.strip(string.punctuation) for w in reference_text.lower().split()]
```
Comment (BrianMouncer) [inline samples/python/console/speech_sample.py:703]: >[w.strip(string.punctuation) for w in reference_text.lower().split()] [](start = 26, length = 69)

This correlate query style of removing the punctuation is amazing, but not easy to follow unless you are familiar with it, or the SQL version of it.

Would it be better for the sample, if we split this into multiple steps that are easy to follow?

File: samples/python/console/speech_sample.py (hunk 1 above)
Comment (yulin-li) [reply samples/python/console/speech_sample.py:700]: thx, updated

File: samples/python/console/speech_sample.py (hunk 2 above)
Comment (yulin-li) [reply samples/python/console/speech_sample.py:703]: it's a good comment, I add some comments to explain this line.

**Review by BrianMouncer — APPROVED** [review]
:shipit:

**Pull request merged**